### ISSUE TITLE
Instructor assessment instance view is missing tab-bar

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#2263]
Clicking "Details" next to an assessment on the "Students" tab takes you to `pages/instructorAssessmentInstance`, but the tab-bar vanishes when you go there. I guess it should still stay with "Students" as the active tab.

**Comment by ZacWarham** [comment]
@mwest1066 is this just changing this

https://github.com/PrairieLearn/PrairieLearn/blob/dd6a70f3fef224645b0fe4befae8f90167aa91ec/apps/prairielearn/src/lib/navPageTabs.ts#L321

To `activeSubPage: ['instances', 'assessment_instance'],`  or am I misunderstanding?

![sample](https://github.com/user-attachments/assets/1175b341-9902-41fe-8126-b3839e4a0931)

**Comment by mwest1066** [comment]
Maybe? CC @miguelaenlle because it might be best to fix this as part of the new nav experience. In any case, @miguelaenlle is the current expert on nav!

**Comment by miguelaenlle** [comment]
Yes, that will resolve the issue! The issue is also present in enhanced nav:

![Image](https://github.com/user-attachments/assets/7208c158-c94c-462a-a18d-ce1d296bef8d)

It is also fixed by changing `activeSubPage: 'instances',` to `activeSubPage: ['instances', 'assessment_instance']`

![Image](https://github.com/user-attachments/assets/80405190-32fb-4385-81fc-3bbbc871f93a)

**Comment by ZacWarham** [comment]
@miguelaenlle - is it worth me adding a PR for this or will you bundle it in your changes?

**Comment by miguelaenlle** [comment]
> [@miguelaenlle](https://github.com/miguelaenlle) - is it worth me adding a PR for this or will you bundle it in your changes?

Hi @ZacWarham . I think it's okay to have this as a separate PR, since I'll be releasing individual feature PRs for each navigation system improvement.
